README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Please, update your readme file to the [Microverse Readme Template](https://github.com/microverseinc/readme-template).   

Remember, all sections are required except for "tests" and "live demo" if not required by the project.

That way, we will meet this particular requirement 💪